src/pages/home/HeaderView.js

```diff
@@ -217,7 +217,7 @@ function HeaderView(props) {
     const shouldShowBorderBottom = !isTaskReport || !isSmallScreenWidth;
     const shouldDisableDetailPage = ReportUtils.shouldDisableDetailPage(props.report);
 
-    const isLoading = !props.report || !title;
+    const isLoading = !props.report || !title || !props.report.reportID;
```

Comment:
```suggestion
    const isLoading = !props.report || !props.report.reportID || !title;
```

NAB. Just looks more readable